frontend/src/host_orchestrator/orchestrator/instancemanager.go

```diff
@@ -129,9 +133,18 @@ type cvdInstance struct {
 	Displays       []string `json:"displays"`
 	InstanceDir    string   `json:"instance_dir"`
 	WebRTCDeviceID string   `json:"webrtc_device_id"`
+	ADBSerial	   string   `json:"adb_serial"`
 }
 
-func (i *cvdInstance) toAPIObject(group string) *apiv1.CVD {
+func (i *cvdInstance) toAPIObject(group string) (*apiv1.CVD, error) {
+	_, port, err := net.SplitHostPort(i.ADBSerial)
```

Comment:
let's put the `ADBSerial` parsing logic into its own function.

Reply:
Removed parsing logic regarding `ADBSerial` in https://github.com/google/android-cuttlefish/pull/522, since HO won't handle it.